paddle/fluid/pybind/protobuf.cc

```diff
@@ -21,77 +21,6 @@ limitations under the License. */
 #include "paddle/fluid/framework/program_desc.h"
 #include "paddle/fluid/framework/var_desc.h"
 
-// Cast boost::variant for PyBind.
```

Comment:
`boost::variant` compatibility has been added at pybind 2.2

Reply:
From the doc, http://pybind11.readthedocs.io/en/latest/advanced/cast/stl.html?highlight=variant#c-17-library-containers  , this feature requires Boost 1.56 or newer. But I ever tried in https://github.com/PaddlePaddle/Paddle/pull/7563, it seems there is some problem. You can try again.